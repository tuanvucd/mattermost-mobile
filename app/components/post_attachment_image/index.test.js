// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import PostAttachmentImage from './index';

describe('PostAttachmentImage', () => {
    const baseProps = {
        height: 100,
        width: 100,
        onError: jest.fn(),
        onImagePress: jest.fn(),
        uri: 'uri',
    };

    it('should match snapshot', () => {
        const wrapper = shallow(<PostAttachmentImage {...baseProps}/>);

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});