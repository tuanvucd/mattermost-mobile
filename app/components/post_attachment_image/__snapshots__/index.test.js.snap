// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostAttachmentImage should match snapshot 1`] = `
<TouchableWithoutFeedback
  onPress={[Function]}
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "justifyContent": "flex-start",
        "marginBottom": 6,
        "marginTop": 10,
      },
      Object {
        "height": 100,
      },
    ]
  }
>
  <View>
    <ForwardRef(forwardConnectRef)
      imageUri="uri"
      onError={[MockFunction]}
      resizeMode="contain"
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderRadius": 3,
            "justifyContent": "center",
            "marginVertical": 1,
          },
          Object {
            "height": 100,
            "width": 100,
          },
        ]
      }
    />
  </View>
</TouchableWithoutFeedback>
`;
